lib/membrane/rtp/session_bin.ex

```diff
@@ -66,13 +66,15 @@ defmodule Membrane.RTP.SessionBin do
 
   An extension will be spawned inside the bin under `{extension_name, ssrc}` name.
 
-  ### Currently supported RTP extensions are:
+  ### RTP plugin ships with the following extensions:
   * `Membrane.RTP.VAD`
   * `Membrane.RTP.TWCCReceiver`
+  * `Membrane.RTP.TWCCSender`
 
   ### Example usage
   `{:vad, %Mebrane.RTP.VAD{vad_id: 1, time_window: 1_000_000}}`
   `{:twcc, %Mebrane.RTP.TWCCReceiver{twcc_id: 1, report_interval: Membrane.Time.milliseconds(250)}}`
+  `{:twcc, Mebrane.RTP.TWCCSender}`
```

Comment:
This isn't rendered properly, I guess this should be a list
```suggestion
  * `{:vad, %Mebrane.RTP.VAD{vad_id: 1, time_window: 1_000_000}}`
  * `{:twcc, %Mebrane.RTP.TWCCReceiver{twcc_id: 1, report_interval: Membrane.Time.milliseconds(250)}}`
  * `{:twcc, Mebrane.RTP.TWCCSender}`
```

Reply:
BTW, the typedoc doesn't seem like a right place for such docs - the examples should rather be in the pad options descriptions